pkgs/applications/emulators/cemu-wiiu/default.nix

```diff
@@ -0,0 +1,134 @@
+{ lib, stdenv, fetchFromGitHub
+, addOpenGLRunpath
+, wrapGAppsHook
+, cmake
+, glslang
+, nasm
+, pkg-config
+
+, SDL2
+, boost
+, cubeb
+, curl
+, fmt_9
+, glm
+, gtk3
+, imgui
+, libpng
+, libzip
+, libXrender
+, pugixml
+, rapidjson
+, vulkan-headers
+, wxGTK32
+, zstd
+
+, vulkan-loader
+}:
+
+let
+  version = "unstable-2022-10-28";
+  rev = "e0aaf631c460cd893c3ec682a237891f2d1baa50";
+  hash = "sha256-GJA/lJJqShuHeYirBW1kyVsU44kMpmAn916PSGOnKkY=";
+
+  # TODO: Maybe put into a separate package? This is written by a Cemu
+  # developer and not really used outside Cemu.
```

Comment:
This package also has `zarchive` tool which could be used for creating `wua` files, which is not easy to do if this package is not exposed.

Reply:
That would indeed be useful to have exposed, I didn't know what it was used for apart from it being a dependency for cemu, so I didn't see a necessity for it to be its own package.